_targets.R

```diff
@@ -591,7 +591,11 @@ list(
         data = episode_file,
         year = year
       )
-    ) # ,
+    ),
+    tar_target(
+      cross_year_tests,
+      process_tests_cross_year(year = year)
+    ), # ,
```

Comment:
I think i need to come back to this as im not sure this will ensure that all newly created ep files will be included in the test comparison